lib/rules/no-control-regex.js

```diff
@@ -51,14 +51,43 @@ module.exports = {
             return null;
         }
 
+        /**
+         * Check the regex expression
+         * @param {String} regexStr regex as string to check
+         * @returns {Boolean} returns true if finds control characters on regex
+         * @private
+         */
+        function checkRegExp(regexStr) {
+
+            // check control characters, if RegExp object used
+
+            var hasControlChars = /[\x00-\x1f]/.test(regexStr); // eslint-disable-line no-control-regex
```

Comment:
Can you combine these two `var` statements? Then you can use `eslint-disable no-control-regex` around both